### PR TITLE
Create SumTwoNumber.cs

### PR DESCRIPTION
**Pull request by DuyVanDev**

### CODE REVIEW

**Comment by TrungThang02** [comment]
amazing !!

File: SumTwoNumber.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
```
Comment (vanthanhdao) [inline SumTwoNumber.cs:6]: //Thieu Using

**Review by vanthanhdao — COMMENTED** [review]
Duyet

File: SumTwoNumber.cs (hunk 1 above)
Comment (DuyVanDev) [reply SumTwoNumber.cs:6]: added

**Pull request merged**